torch/profiler/_memory_profiler.py

```diff
@@ -97,7 +105,6 @@ class TensorKey:
 
     id: int
     storage: _Storage
-    device: torch.device
 
     def __repr__(self) -> str:
         return f"id={self.id}: {repr(self.storage):<24} ({self.device})"
```

Comment:
probably need to remove self.device here?

Reply:
Since we are inheriting self.device from Key, is the better idea to make a __repr__ in class Key?

Reply:
ah I missed that, it looks fine then!